Godeps/_workspace/src/github.com/spf13/pflag/flag.go

```diff
@@ -289,6 +290,16 @@ func (f *FlagSet) MarkDeprecated(name string, usageMessage string) error {
 	return nil
 }
 
+// Mark the shorthand of a flag deprecated in your program
+func (f *FlagSet) MarkShorthandDeprecated(name string, usageMessage string) error {
```

Comment:
Code updated manually. It's easier to test the code change here. @deads2k @eparis please take a look and I'll apply this change to pflag package.